docs/apis/rta/s2s_events_api.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Server to server events - overview"
```

Comment:
title is the title of the specific page. Call it only "overview". The rest redundant